src/Pages/Main/Components/SearchSpaceList.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react'
+import "./SearchSpaceList.scss"
+
+export default class SearchSpaceList extends Component {
+ 
+  render() {
+    return (
+      <div className="SearchSpaceList">
+         <li className="listContents">
+           {this.props.spaceListFromMain}
```

Comment:
간결하게 보일 수 있게 비구조화 해주세요!
```suggestion
render() {
  const {spaceListFromMain} = this.props;
  return (
    ...
           {spaceListFromMain}
```